src/envoy/mixer/grpc_transport.cc

```diff
@@ -72,6 +82,18 @@ bool GrpcTransport::IsMixerServerConfigured(Upstream::ClusterManager& cm) {
   return cm.get(kMixerServerClusterName) != nullptr;
 }
 
+void GrpcTransport::onPreRequestCustomizeHeaders(Http::HeaderMap& headers) {
+  if (!headers_) return;
+
+  for (const auto& key : kHttpTraceHeaders) {
+    const HeaderEntry* entry = headers_->get(key);
```

Comment:
I did not realized get() is NOT map lookup.  I am changing code to use inline_headers directly.